circuit-conversations-list.js

```diff
@@ -0,0 +1,210 @@
+const template = document.createElement('template');
+const initHtml = '<select class="select-wide" part="conversation-list" id="conversations-list"></select>';
+template.innerHTML = initHtml;
```

Comment:
Why don't you just use the same as the other components by assigning it directly?